### PR TITLE
Utilise Eddsa & Public Keys - Part 3

### PR DESCRIPTION
**Pull request by biscuitdey**
# Description

Use Eddsa signatures in transactions, workgroups, state and builderData

## Related Issue

#759 

## Motivation and Context

Use multiple types of signatures 

## How Has This Been Tested

Running all existing tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts [hunk 1]
```diff
@@ -76,17 +78,49 @@ beforeEach(async () => {
...
   });
 
   bpiSubject2 = await prisma.bpiSubject.create({
     data: {
       name: 'name2',
       description: 'desc2',
-      publicKey:
-        '0x04203db7d27bab8d711acc52479efcfa9d7846e4e176d82389689f95cf06a51818b0b9ab1c2c8d72f1a32e236e6296c91c922a0dc3d0cb9afc269834fc5646b980',
+      publicKeys: {
+        create: [
```
Comment (Therecanbeonlyone1969) [inline examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts:108]: @biscuitdey why is this `create` and in L82 it is `createMany`?

**Review by Therecanbeonlyone1969 — DISMISSED** [review]
One minor nit / question but that should not hold up merging if all tests work.

File: examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts (hunk 1 above)
Comment (biscuitdey) [reply examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts:108]: Fixed.

**Comment by biscuitdey** [comment]
Code added to #794

**Pull request closed without merging**